discord/ext/commands/cooldowns.py

```diff
@@ -297,7 +297,7 @@ class _Semaphore:
 
     def __init__(self, number: int) -> None:
         self.value: int = number
-        self.loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
+        self.loop: asyncio.AbstractEventLoop = asyncio.get_running_loop()
```

Comment:
This change will break decorators.

Reply:
even if I change this back to `get_event_loop`, in future Python versions when `get_event_loop` will act as an alias to `get_running_loop`, this will break, so I think we need a better solution

Reply:
Actually, I double checked and this change is fine since these are only created within an async function.